lib/pacemaker/pcmk_sched_location.c

```diff
@@ -165,16 +165,11 @@ generate_location_rule(pcmk_resource_t *rsc, xmlNode *rule_xml,
         case pcmk__combine_or:
             break;
 
-        default:
-            /* @COMPAT When we can break behavioral backward compatibility,
-             * return false
-             */
-            pcmk__config_warn("Location constraint rule %s has invalid "
-                              PCMK_XA_BOOLEAN_OP " value '%s', using default "
-                              "'" PCMK_VALUE_AND "'",
+        default: // Not possible with schema validation enabled
+            pcmk__config_warn("Ignoring location constraint rule %s "
```

Comment:
Should this become `pcmk__config_err()`?

Reply:
yes